config/jobs/kubernetes/sig-testing/bazel-build-test.yaml

```diff
@@ -222,9 +222,9 @@ periodics:
             --config=ci \
             "--build=//... -//vendor/..." \
             --release=//build/release-tars \
-            --gcs=gs://kubernetes-release-dev/ci-periodic \
+            --gcs=gs://kubernetes-release-dev/ci \
```

Comment:
I'm not sure I have enough context to understand this change, what's the reasoning here?